client/lib/wp/handlers/test/guest-sandbox-ticket.js

```diff
@@ -65,7 +65,7 @@ describe( 'guest-sandbox-ticket', () => {
 
 			injectGuestSandboxTicketHandler( wpcom );
 
-			expect( wpcom.request( { query: 'search=whatever' } ) );
```

Comment:
`wpcom.request` calls expect in this test. This call is misleading / not required.